### ISSUE TITLE
Add support for `python3 -m autoimport` invocation

### ISSUE THREAD

**Issue by giampaolo** [lyz-code/autoimport#210]
## Description

Python CLI tools typically give the ability to be invoked via `python3 -m <NAME>`. This is useful when multiple python versions are installed, and one want so pick up a specific version. `isort`, `flake8`, `black` and others all support this functionality. `autoimport` does not:

```
~$ python3 -m autoimport
/usr/local/bin/python3: No module named autoimport.__main__; 'autoimport' is a package and cannot be directly executed
```

## Possible Solution
This can be easily achieved by adding a `__main__.py` file in the same directory as `__init__.py`. Within `__main__.py` it should be sufficient to just do (not tested):

```python
from .entrypoints.cli import cli
cli()
```

**Comment by lyz-code** [comment]
Hi @giampaolo, thanks for taking the time to open the issue. 

I see your point and if you want to make a PR with the changes we'll merge it, sadly implementing this won't be in the top of my priority list

**Comment by lyz-code** [comment]
Available since `1.3.0` thanks to @ErikBjare
